ssn/index.html

```diff
@@ -1513,9 +1513,14 @@ <h5><a href="#SOSAActuation">sosa:Actuation</a></h5>
           <p><strong>an OWL Class</strong></p>
 
           <em property="rdfs:label">Actuation</em> -
-          <span property="rdfs:comment skos:definition">An <a href="#SOSAActuation"><code>Actuation</code></a> carries
-            out an (Actuation) <a href="#SOSAProcedure"><code>Procedure</code></a> to change the state of the world
-            using an <a href="#SOSAActuator"><code>Actuator</code></a>.</span> <br>
+          <span property="skos:definition">An <a href="#SOSAActuation"><code>Actuation</code></a> carries out an
+            (Actuation) <a href="#SOSAProcedure"><code>Procedure</code></a> to change the state of the world using an <a
+              href="#SOSAActuator"><code>Actuator</code></a>. <a href="#SOSAhasResult"><code>hasResult</code></a>
+            indicates the intended value of the <a href="#SOSAActuatableProperty"><code>ActuatableProperty</code></a> at
```

Comment:
> the ActuatableProperty

What if there are multiple? How can we differentiate between them?

Reply:
As [mentioned in the other thread](https://github.com/w3c/sdw-sosa-ssn/issues/112#issuecomment-1899103582) that was news to me. 
Clearly this is an issue that needs to be addressed. 

My preferred approach is to limit cardinality to 1, and push the question of correlating result and property vectors into future work.

Reply:
I agree that we should keep the same pattern as Observation and set the cardinality to exactly ONE.

Although we have applications of SOSA/SSN where we use result and property vectors, I'm not sure this should be in the scope of the ontology. Applications can define their own mechanisms to handle this (as we do in connected systems).

Reply:
Establishing ActuationCollection would provide a standard pathway for applications with multiple ActuatableProperties and Results. 
I'll have a go at that.

Reply:
OK - I've added ActuationCollection in this branch